drivers/netif-driver.c

```diff
@@ -78,6 +78,9 @@ static err_t netif_dev_init(struct netif *netif)
 		netif->hwaddr_len = ETH_HWADDR_LEN;
 		netif->flags = NETIF_FLAG_UP | NETIF_FLAG_LINK_UP | NETIF_FLAG_BROADCAST |
 		NETIF_FLAG_ETHARP | NETIF_FLAG_ETHERNET;
+#if LWIP_IPV6
```

Comment:
should be under ` LWIP_IPV6 && LWIP_IPV6_MLD` define?

Reply:
done